week3/61-sistiema-linieinykh-uravnienii-1/task.py

```diff
@@ -0,0 +1,20 @@
+def safe_to_int(n):
+    if n.is_integer():
+        return int(n)
+    else:
+        return n
+
+
+def main():
+    a, b, c, d, e, f = float(input()), float(input()), float(
```

Comment:
use variables from Task!

code unreadable - read 3 variables on first line and other 3 on second

Reply:
done